AU2/frontends/inquirer_cli.py

```diff
@@ -472,12 +472,14 @@ def render(html_component, dependency_context={}):
         return {}
 
     elif isinstance(html_component, Checkbox):
+        default = "Yes" if html_component.checked else "No"

```

Comment:
nit - I think I prefer having "yes" and "no" near each other in this case, although not super fussed

```suggestion
        default = html_component.checked and "Yes" or "No"
```

if you've not seen this style before, it looks like `default` will be a `bool`, but it's actually a `str`

```
>>> True and "Yes" or "No"
'Yes'
>>> False and "Yes" or "No"
'No'
```